app/data/institutions.ts

```diff
@@ -3347,4 +3347,3544 @@ export const institutions: Institution[] = [
     supportedPayment: ["duitnow", "tng"],
     coords: [6.352203154708551, 99.88020500425073],
   },
+  {
+    id: 279,
+    name: "Surau As Sakinah",
+    category: "surau",
+    state: "W.P. Kuala Lumpur",
+    city: "Kuala Lumpur",
+    qrImage: "",
+    qrContent: "00020201021126610014A000000615000101065641670215QRMID0000000001031001230919085204866153034585802MY5916SURAU AS SAKINAH6012KUALA LUMPUR6105501006304B99A",
+    supportedPayment: ["duitnow", "tng"],
+  },
+  {
+    id: 280,
+    name: "Masjid Damansara Perdana",
+    category: "mosque",
+    state: "Selangor",
+    city: "Petaling Jaya",
+    qrImage: "",
+    qrContent: "00020201021126610014A000000615000101065641670215QRMID0000005045031001338839665204566153034585802MY5924MASJID DAMANSARA PERDANA6013PETALING JAYA6105478206304EFB2",
+    supportedPayment: ["duitnow", "tng"],
+  },
+  {
+    id: 281,
+    name: "Masjid Angkatan Tentera",
+    category: "mosque",
+    state: "W.P. Kuala Lumpur",
+    city: "Sungai Besi",
+    qrImage: "",
+    qrContent: "00020201021126610014A000000615000101065641670215QRMID0000000297031001221777205204866153034585802MY5923MASJID ANGKATAN TENTERA6011SUNGAI BESI61055700063043EC8",
+    supportedPayment: ["duitnow", "tng"],
+  },
+  {
+    id: 282,
+    name: "Surau Nurul Iman",
+    category: "surau",
+    state: "W.P. Kuala Lumpur",
+    city: "Taman Minang Kl",
+    qrImage: "",
+    qrContent: "00020201021126610014A000000615000101065641670215QRMID0000000318031001692066535204866153034585802MY5916SURAU NURUL IMAN6015TAMAN MINANG KL61055600063048622",
+    supportedPayment: ["duitnow", "tng"],
+  },
+  {
+    id: 283,
+    name: "Masjid Nurrusaadah ",
+    category: "mosque",
+    state: "W.P. Kuala Lumpur",
+    city: "Kuala Lumpur",
+    qrImage: "",
+    qrContent: "00020201021126610014A000000615000101065641670215QRMID0000000391031001622145835204866153034585802MY5919MASJID NURRUSAADAH 6012KUALA LUMPUR6105561006304CE8E",
+    supportedPayment: ["duitnow", "tng"],
+  },
+  {
+    id: 284,
+    name: "Masjid Al Mukhlisin",
+    category: "mosque",
+    state: "W.P. Kuala Lumpur",
+    city: "Cheras",
+    qrImage: "",
+    qrContent: "00020201021126610014A000000615000101065641670215QRMID0000000549031001391372065204866153034585802MY5919MASJID AL MUKHLISIN6006CHERAS6105560006304613C",
+    supportedPayment: ["duitnow", "tng"],
+  },
+  {
+    id: 285,
+    name: "Surau Al-Mukhlisin",
+    category: "surau",
+    state: "W.P. Kuala Lumpur",
+    city: "Cheras",
+    qrImage: "",
+    qrContent: "00020201021126610014A000000615000101065641670215QRMID0000001196031001238230665204866153034585802MY5918SURAU AL-MUKHLISIN6006CHERAS6105560006304797E",
+    supportedPayment: ["duitnow", "tng"],
+  },
+  {
+    id: 286,
+    name: "Surau At-Taqwa",
+    category: "surau",
+    state: "Kedah",
```

Comment:
state kedah but city cheras

Reply:
seems to be an edge case here when processing the postcode via the script should be state W.P. Kuala Lumpur checked in pdf